keyboards/cu80/cu80.h

```diff
@@ -0,0 +1,80 @@
+	K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212,       K214, K215, K216, K217, \
+	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313,       K315, K316, K317, \
+	K400,       K402, K403, K404, K405, K406, K407, K408, K409, K410, K411, K412, K413, K414,                   \
+	K500, K501, K502, K503, K504, K505, K506, K507, K508, K509, K510, K511,       K513,             K516,       \
+	K600, K601,       K603,             K606,                   K610, K611, K612,       K614, K615, K616, K617  \
+) { \
+	{ K000,  K001,  K002,  K003,  K004,  K005,  K007,  K008,  K009,  KC_NO, K011,  K012,  K013,  K014,  K015,  K016,  K017 }, \
+	{ K200,  K201,  K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K214,  K215,  K216,  K217 }, \
+	{ K300,  K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K313,  KC_NO, K315,  K316,  K317 }, \
+	{ K400,  K402,  K403,  K404,  K405,  K406,  K407,  K408,  K409,  K410,  K411,  K412,  K413,  K414,  KC_NO, KC_NO, KC_NO }, \
+	{ K500,  K501,  K502,  K503,  K504,  K505,  K506,  K507,  K508,  K509,  K510,  K511,  KC_NO, K513,  KC_NO, K516,  KC_NO }, \
+	{ K600,  K601,  K603,  KC_NO, KC_NO, KC_NO, K606,  KC_NO, KC_NO, K610,  K611,  K612,  KC_NO, K614,  K615,  K616,  K617 }  \
+}
+
+#define LAYOUT_iso( \
+	K000, K001, K002, K003, K004, K005,       K007, K008, K009,       K011, K012, K013, K014, K015, K016, K017, \
+	                                                                                                            \
+	K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212,       K214, K215, K216, K217, \
+	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313,       K315, K316, K317, \
+	K400,       K402, K403, K404, K405, K406, K407, K408, K409, K410, K411, K412, K413, K414,                   \
+	K500, K501, K502, K503, K504, K505, K506, K507, K508, K509, K510, K511,       K513,             K516,       \
+	K600, K601,       K603,             K606,                   K610, K611, K612,       K614, K615, K616, K617  \
+) { \
+	{ K000,  K001,  K002,  K003,  K004,  K005,  K007,  K008,  K009,  KC_NO, K011,  K012,  K013,  K014,  K015,  K016,  K017 }, \
+	{ K200,  K201,  K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K214,  K215,  K216,  K217 }, \
+	{ K300,  K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K313,  KC_NO, K315,  K316,  K317 }, \
+	{ K400,  K402,  K403,  K404,  K405,  K406,  K407,  K408,  K409,  K410,  K411,  K412,  K413,  K414,  KC_NO, KC_NO, KC_NO }, \
+	{ K500,  K501,  K502,  K503,  K504,  K505,  K506,  K507,  K508,  K509,  K510,  K511,  KC_NO, K513,  KC_NO, K516,  KC_NO }, \
+	{ K600,  K601,  K603,  KC_NO, KC_NO, KC_NO, K606,  KC_NO, KC_NO, K610,  K611,  K612,  KC_NO, K614,  K615,  K616,  K617 }  \
+}
```

Comment:
It's the same as `LAYOUT_all`, so we can alias it. Also renaming so we can support [Community Layouts](https://docs.qmk.fm/#/feature_layouts).

```suggestion
#define LAYOUT_tkl_iso LAYOUT_all
```

Reply:
Cheers, I should have been more aware of the community layouts!